chia/daemon/client.py

```diff
@@ -19,11 +19,13 @@ def __init__(
         uri: str,
         ssl_context: Optional[ssl.SSLContext],
         max_message_size: int = 50 * 1000 * 1000,
+        heartbeat: int = 300,
```

Comment:
Does this repetition of the default add something?  Or should it just be a required parameter here.

Reply:
doesn't seem to hurt anything to have it

Reply:
I changed this , it's required parameter now